### PR TITLE
Create NTN

### PR DESCRIPTION
**Pull request by AudreyIATI**
HI @kd-ods,

This is related to ongoing conversation [here](https://github.com/org-id/register/issues/441)

Please let me know if I need to do anything else.

Thanks,
Audrey

### CODE REVIEW

**Review by kd-ods — CHANGES_REQUESTED** [review]
Hi @AudreyIATI - there are a few things to look at here. It will probably be easiest for you to edit the file back in the editing app. I've added details of how to do that at the end of the Guidance doc.

File: lists/PK/pk-ntn.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": {
+    "en": "Pakistan National Tax Number",
+    "local": "English"
+  },
+  "url": "https://e.fbr.gov.pk/esbn/Verification#",
+  "description": {
+    "en": "This list is a record from the Federal Board of Revenue, Government of Pakistan. It contains active, blacklisted and exempt Taxpayer Lists based on the National Tax Number (NTN) of organisations.\nIt also contains Customs information.\nOrganisations registered with the Inland Revenue in Pakistan should be found on this list."
```
[the remark this answers is not in the document]
Comment (AudreyIATI) [reply lists/PK/pk-ntn.json:8]: Done!

File: lists/PK/pk-ntn.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  },
+  "coverage": [
+    "PK"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
```
[the remark this answers is not in the document]
Comment (AudreyIATI) [reply lists/PK/pk-ntn.json:14]: Done!

File: lists/PK/pk-ntn.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    "company/community_interest_company"
+  ],
+  "sector": [
+    "humantiarian_relief"
```
[the remark this answers is not in the document]
Comment (AudreyIATI) [reply lists/PK/pk-ntn.json:18]: Done!

File: lists/PK/pk-ntn.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  ],
+  "code": "PK-NTN",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "primary",
```
[the remark this answers is not in the document]
Comment (AudreyIATI) [reply lists/PK/pk-ntn.json:23]: Done!

File: lists/PK/pk-ntn.json [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    "availableOnline": true,
+    "onlineAccessDetails": null,
+    "publicDatabase": "https://e.fbr.gov.pk/esbn/Verification#:",
+    "guidanceOnLocatingIds": "From homepage use the drop down on the left to choose Active Taxpayer List (income tax)\nParameter type: NTN\nEnter Registration number\nVerify\nNote - National Tax Number format is 4174941-3\nThe number after the dash is the check number - remove this when you search.",
+    "exampleIdentifiers": "4174941-3",
+    "languages": ["en"]
+  },
+  "data": {
+    "availability": [
+      "pdf"
```
[the remark this answers is not in the document]
Comment (AudreyIATI) [reply lists/PK/pk-ntn.json:34]: The ATL output gives a pitiful amount of information (shown below), the bulk export provides an Excel sheet with every registered NTN, but no additional information beyond the below.

Registration No
Organisation name
Business name
Filing status
Filing status checking date

**Comment by AudreyIATI** [comment]
Hi @kd-ods = changes made, and new json pasted over the old.
Let me know if I need to change/correct anything.

Thanks!
Audrey

**Comment by AudreyIATI** [comment]
Hi @kd-ods - checking to see if there's any update on this?

**Review by kd-ods — APPROVED** [review]
This looks great, Audrey. I realise it took some time for me to get to this. Thanks for your patience.

I'll merge this and update the website now.

**Comment by AudreyIATI** [comment]
No worries @kd-ods - Thank you for all your help!

**Pull request merged**